docs/sources/setup/install/helm/deployment-guides/aws.md

```diff
@@ -0,0 +1,763 @@
+---
+title: Deploy the Loki Helm chart on AWS
+menuTitle: Deploy on AWS
+description: Installing the Loki Helm chart on AWS.
+keywords: 
+---
+
+# Deploy the Loki Helm chart on AWS
+
+This guide shows how to deploy a minimally viable Loki in either **microservice** or **monolithic** mode on AWS using the Helm chart. To run through this guide, we expect you to have the necessary tools and permissions to deploy resources on AWS, such as:
+
+- Full access to EKS (Amazon Elastic Kubernetes Service)
+- Full access to S3 (Amazon Simple Storage Service)
+- Sufficient permissions to create IAM roles and policies
+
+There are two methods for authenticating and connecting Loki to AWS S3. We will guide you through the recommended method of granting access via an IAM role.
+
+## Prerequisites
+
+- Helm 3 or above. Refer to [Installing Helm](https://helm.sh/docs/intro/install/). This should be installed on your local machine.
+- A running Kubernetes cluster on AWS. A simple way to get started is by using EKSctl. See [Getting started with EKSctl](https://eksctl.io/).
+- Kubectl installed on your local machine. Refer to [Install and Set Up kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/).
+- (Optional) AWS CLI installed on your local machine. Refer to [Installing the AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/install-cliv2.html). This is required if you plan to use EKSctl to create the EKS cluster and modify the IAM roles and policies locally.
+  
+### EKS Minimum Requirements
+
+{{< admonition type="caution" >}}
+These EKS requirements are the minimum specification needed to deploy Loki using this guide. You may wish to adjust plugins and instance types based on your AWS environment and workload. **If you choose to do so, this may invalidate the guide.**
```

Comment:
Since you said this was a guide for BOTH monolithic and microservices, I think this admonition is going to be confusing for users.  Is this the minimum specification for monolithic?   OR for microservices?